### ISSUE TITLE
Official channels

### ISSUE THREAD

**Issue by CIAvash** [perl6/user-experience#32]
There is an issue with not announcing things through official channels. Rakudo has its own blog. But Perl 6 doesn't. Maybe it should?

For example [the Perl 6.d announcement](http://blogs.perl.org/users/zoffix_znet/2018/11/announce-raku-perl-6-diwali-6d-language-specification-release.html) was done by Zoffix on blogs.perl.org with his own account, and the official twitter account simply retweeted Zoffix's tweet.

This can be problematic, there are people who think/thought Zoffix is doing things on his own, specially aliasing(which might also be related to #31).

I think we should make better use of our official channels and if we don't have them(blog), create them.

**Comment by stmuk** [comment]
I don't see this a useful issue. It's a thinly disguised attack on a dedicated key member of the perl 6 team over a minor petty point -- the exact way the announcement was made.  

Perl6 isn't about red tape and process we are hackers (in the broader, older sense). Remember, too that this is someone who spent a huge amount time on working on 6.d. What message are we sending to future volunteers with this?

I propose this issue be closed.  This is a misuse of github which should be used for specific technical issue rather than general politics.

**Comment by CIAvash** [comment]
What?! It's actually in defense of Zoffix! Read [this](https://twitter.com/andrewshitov/status/1059389501318090752) and [this](https://nulled.red/@flussence/101023350804911046) for a few examples.

Also if we had a blog.perl6.org, then Zoffix could write it with his own account. I'm not sure what you're reading.

**Comment by JJ** [comment]
I'm not sure there's an official Twitter account. Is that the case? I'm
sure there are extraofficial official ones, but nothing like an "official"
one.
WRT the OP, that's not a clarification we are able to make, or should make.
I think it's up to @timtoady to point us in the right direction.

**Comment by virtualsue** [comment]
I did *thumbs down*  because this doesn't seem to have any relevance to anything on github. If you're bikeshedding about how the release is announced, either help by setting up the accounts you think should exist, or just talk about it on irc.

**Comment by stmuk** [comment]
@CIAvash I can't see anything of interest in either of your two links. If you want to create a blog.perl6.org please go ahead. We are volunteers not bureaucrats and "official channels" means nothing. Anyway I'm out of this thread.

**Comment by CIAvash** [comment]
@JJ https://twitter.com/perl6org

If there are no official things, then what are perl6.org and the perl6 group on github?

@virtualsue It's about Perl 6 user experience, what does anything on github have to do with it?

**Comment by coke** [comment]
I disagree with @virtualsue and @stmuk here, I think this is a fine place for a question about how we announce things.

**Comment by JJ** [comment]
El mar., 6 nov. 2018 a las 14:57, Siavash Askari Nasr (<
notifications@github.com>) escribió:

> @JJ <https://github.com/JJ> https://twitter.com/perl6org
>
> If there are no official things, then what are perl6.org and the perl6
> group on github?
>

Well, those are official, but also collective, in the sense that anyone
with write access to the repo can publish there (and anyone else review). I
might have missed it, or maybe I just don't know where those things are
made, but I think that account is extraofficial. It's not even listed in
perl6.org as the official twitter account, and it's actually called "Perl 6
news feed", not perl6 official account or whatever. I might be wrong,
however. It's true that I don't know who that account belongs to, but it's
just me, maybe everyone else knows.

JJ

**Comment by zoffixznet** [comment]
> It's a thinly disguised attack

I didn't read it that way.

> For example the Perl 6.d announcement was done by Zoffix on blogs.perl.org with his own account, and the official twitter account simply retweeted Zoffix's tweet.

Yeah, and it was almost done on `perl6.party` with [the dancing cats](https://perl6.party/about) on it, before I remembered I had a more appropriate blog space for it.

It would be *very* helpful to have some central communications channels the core team has access to to put out announcements without limiting it to some mailing list (similar to how there is alerts.perl6.org the release managers use to disseminate critical alerts).

**Comment by CIAvash** [comment]
A Jekyll blog could also work, but then the problem will be the design and if it needs to be consistent with other things.

So maybe what you said is better(at least until the media group becomes a thing maybe).

It's good to have a place that people know to look up when they need to find announcements and also subscribe to its feed(which blogs.perl.org has) if they want to follow it.

About the social media accounts, they are mentioned on [Perl 6 Resources](https://perl6.org/resources/), but that doesn't mean they are official. So I don't know about that, but it would be nice maybe in addition to retweeting(or re-posting) individuals' posts, have a separate post for announcements.

**Comment by JJ** [comment]
El mié., 7 nov. 2018 a las 10:12, Siavash Askari Nasr (<
notifications@github.com>) escribió:

> A Jekyll blog could also work, but then the problem will be the design and
> if it needs to be consistent with other things.
>
> So maybe what you said is better(at least until the media group becomes a
> thing maybe).
>
> It's good to have a place that people know to look up when they need to
> find announcements and also subscribe to its feed(which blogs.perl.org
> has) if they want to follow it.
>
> About the social media accounts, they are mentioned on Perl 6 Resources
> <https://perl6.org/resources/>, but that doesn't mean they are official.
> So I don't know about that, but it would be nice maybe in addition to
> retweeting(or re-posting) individuals' posts, have a separate post for
> announcements.
>
>
Well, it would be nice to have an official Twitter account that would do
that, at least...

**Comment by CIAvash** [comment]
Yeah, and if nobody wants to write the post manually, something like IFTTT can be used to tweet the feed items automatically.

**Comment by virtualsue** [comment]
> @JJ https://twitter.com/perl6org
> 
> If there are no official things, then what are perl6.org and the perl6 group on github?
> 
> @virtualsue It's about Perl 6 user experience, what does anything on github have to do with it?

I don't agree that this is a Perl 6 user experience issue. The "alternative narrative" (I'd call it a lie) that Zoffix is "doing things on his own" has been promulgated by certain people behaving IMO in a destructive manner. 

I also don't believe that an official account would have helped in this particular instance, although Zoffix or anyone else might find it handy to keep a separate account for announcement purposes - in which case, just register one. Moritz has a Perl 6 News Feed twitter account, for example. Steve runs the RSS aggregator. People just pitch up and do things.

As a counter example, just the other day a conference organiser was using what is meant to be the 'official account' for the next European Perl conference to voice his own angry opinions and threaten to call off the conference (nb I believe this threat has since been rescinded). I thought it looked worse that the discussion happened via an account set aside for the conference. If it had been coming directly from the person who was upset, then it would have been more honest and open.

**Comment by abraxxa** [comment]
Note that although I'm in the #perl6 IRC channel every workday and have subscribed the 'Weekly changes in and around Perl 6' RSS feed from https://p6weekly.wordpress.com I haven't noticed this dispute until this github issue was opened.
I doubt that anyone outside the core Perl 6 bubble took notice, same for the childish naming alias discussions.

**Comment by CIAvash** [comment]
Maybe by including the example I put a lot of emphasis on it, but that wasn't the main reason, it was just an example(another example happened yesterday, but I'm not going to link to it). Also about "lie", I went with not assuming the worst intentions(which I think is another problem of the community).

The main thing is having a central blog for announcements, the social media accounts come after that which can be solved by automatically posting the blog's feed as I mentioned in another comment.

I guess this is another example of miscommunications.

**Comment by virtualsue** [comment]
I refer you to my original comment about bikeshedding. That's what this is.

**Comment by lucasbuchala** [comment]
Hello @CIAvash. Hopefully I understood the intention of this issue and I completely agree that there should be a central blog for this project (served at `blog.perl6.org`).

FWIW, I think that a good solution would be a **Jekyll** blog, hosted in GitHub, with Markdown files editable by the authors. This new blog should be the place for:

* Weekly reports
* Advent calendar posts
* Guest posts
* New releases announcements (Rakudo Star, + others)

With a Jekyll blog, it should be easy to setup a consistent layout for all posts with a Perl 6 theme/colorscheme.

**Comment by lizmat** [comment]
Sorry, I didn't look carefully enough: I thought I was closing #33 . My apologies.

**Comment by chsanch** [comment]
I think is related to [Perl 6 at a glance ](https://github.com/perl6/perl6.org/issues/119)

**Comment by zoffixznet** [comment]
> almost done on perl6.party with the dancing cats

To add another point to OP: there has been confusion in my Twitter feed between my announcing 6.d release and my personal use of "Raku" as language name alias. So another benefit to official channels is clearer separation between official messaging and developers' personal viewpoints.

**Comment by AlexDaniel** [comment]
@CIAvash can you submit a new ticket in the [problem-solving](https://github.com/perl6/problem-solving/) repo? I think the situation is significantly different now.
